tick/array/tests/serialize_test.py

```diff
@@ -60,6 +60,22 @@ def test_serialize_sparse_2d_array(self):
         # python 3.5 has show to required this - investigate typemappers
         gc.collect()
 
+    def test_serialize_column_major_sparse_2d_array(self):
+        """...Test serialization of 2d sparse array is done as expected
+        """
+        array = sparse.rand(10, 10, density=0.3,
+                            format='csc').astype(self.dtype)
+        serialize_array(array, self.array_file)
+
+        serialized_array = load_array(self.array_file, array_dim=2,
+                                      array_type='sparse', dtype=self.dtype,
+                                      format="csc")
+        np.testing.assert_array_almost_equal(array.toarray(),
+                                             serialized_array.toarray())
+
+        # python 3.5 has show to required this - investigate typemappers
+        gc.collect()
+
```

Comment:
Shouldn't we add a test for column major dense array?

Reply:
there's a gtest, we can move it to python